src/formik.tsx

```diff
@@ -91,14 +95,20 @@ export interface FormikState<V> {
 export interface FormikActions<P, V> {
   /* Manually set top level error */
   setError: (e: any) => void;
-  /* Manually set Errors */
+  /* Manually set errors object */
   setErrors: (errors: FormikErrors) => void;
   /* Manually set isSubmitting */
   setSubmitting: (isSubmitting: boolean) => void;
-  /* Manually set touched fields */
+  /* Manually set touched object */
   setTouched: (touched: FormikTouched) => void;
-  /* Manually set values  */
+  /* Manually set values object  */
   setValues: (values: V) => void;
+  /* Set value of form field directly */
+  setFieldValue: (field: string, value: any) => void;
```

Comment:
The reason we got into this whole `setXThing` in the first place is because Formik 0 was built with recompose which create an HOC and an updater function for each state key. This is no . longer needed